scripts/extension-version.js

```diff
@@ -3,80 +3,83 @@ const semVer = require('semver')
 function nextExtensionVersion({
```

Comment:
Add test cases for the changes in this function. Do the existing tests pass?

Reply:
no, they don't currently, I'll update those 👍

Reply:
fixed the tests 👍